elastic/kibana
# [Fleet] Mark shipper feature as experimental in Fleet UI

Fixes https://github.com/elastic/kibana/issues/153845

## Summary

The new Outputs Shipper section will be available as an experimental feature in 8.8.0. This PR adds a badge to clearly indicate it as experimental in UI.



To test it:
Enable the shipper options by adding one of the following in outputs yaml editor:

```
shipper: {}
```
OR
```
shipper:
  enabled: true
```

You should see a badge besides "Advanced options":
<img width="734" alt="Screenshot 2023-04-03 at 12 13 37" src="https://user-images.githubusercontent.com/16084106/229481384-cce7d105-1e03-4e97-9815-a7cef3a29d11.png">
<img width="737" alt="Screenshot 2023-04-03 at 12 13 30" src="https://user-images.githubusercontent.com/16084106/229481485-ff963b86-da9d-4f8e-ab23-277f7f9290f6.png">
